README.md

```diff
@@ -77,21 +64,15 @@ property or address. There are three geography<sup>*</sup> fields and one field
 ## API Specification
 The API you will be implementing for this project must adhere to the following API specification:
 
-### GET /display/:id?(overlay=yes(&parcel=:parcelColor)(&building=:buildingColor))
+### GET /display/:id
 
-*Fetches and displays property tile by ID. Optionally overlays parcel and building geometries on tile.*
+*Fetches and displays property tile by ID.*
 
-`example: GET localhost:1235/display/f853874999424ad2a5b6f37af6b56610?overlay=yes&building=green&parcel=orange`
+`example: GET localhost:1235/display/f853874999424ad2a5b6f37af6b56610`
```

Comment:
Is this what will return the geometries for showing in the map? If so, do we want a different url other than `display`?

Reply:
Oh, i think i see the properties API below, just not part of the diff so hard to see. So that'll likely be the thing that will return the geometries.